execution/executor/src/lib.rs

```diff
@@ -186,8 +186,10 @@ where
         let first_txn_version = match txn_list_with_proof.first_transaction_version {
             Some(tx) => tx as Version,
             None => {
-                error!(StructuredLogEntry::new_named("MUST_FIX", "assertion")
-                    .data("details", "first_transaction_version should exist."));
+                error!(
+                    category = "MUST_FIX",
+                    "first_transaction_version should exist"
+                );
                 return Err(anyhow!("first_transaction_version should exist."));
```

Comment:
I'm kinda surprised this isn't an expect, since we cannot recover, can we? @lightmark

Reply:
It's usually the issue of input so executor shouldn't crash. cc@mimoo

Reply:
If it doesn't crash, then we have a useless node... does the error propagate upward to an expect statement?